### DISCUSSION TITLE
Could the message after exporting be suppressed?

### DISCUSSION THREAD

**Discussion by stvhuang** [nvim-orgmode/orgmode#357] [Q&A] [answered]
<img width="829" alt="image" src="https://user-images.githubusercontent.com/15218222/176386500-c397e22c-3b28-4772-bc38-34d0eacbc1b2.png">

Hi, as title, can this message be disabled?
I would like to manually open the exported file so this prompt is kinda useless for me. Thus I wanna disable it.

**Comment by jgollenz** [comment, marked as answer]
There is no option that I'm aware of.
